submissions/medinatapanpa/html/dufuna.html

```diff
@@ -1,51 +1,43 @@
 <!DOCTYPE html>
```

Comment:
Please move all files for this project to the same base folder. Either **html** or **landingPage**; not both

Reply:
Please add a link to the registration page on the home page

Reply:
i have liked the registration page and moved all files to the landing folder.